packages/components/CHANGELOG.md

```diff
@@ -15,6 +11,99 @@
 
 ### Bug Fixes
 
```

Comment:
It seems that the changelog got messed while doing the cherry pick. As this will release a minor package. How should we update it?

Reply:
It seems there have been similar cases in the past:

- https://github.com/WordPress/gutenberg/pull/65905
- https://github.com/WordPress/gutenberg/pull/65853

How about manually adding [just these two entries](https://github.com/WordPress/gutenberg/pull/67258/files#diff-b5aa808176801bfde2277ac886bbbdf0863de109950303cdb89e2d33ebe16a28R22-R23) to the Bug Fixes section?

Reply:
I did it in https://github.com/WordPress/gutenberg/pull/67258/commits/0bc540ddec34913e8e36241958c5813d98482696